### PR TITLE
Create 404/500 pages

### PR DESCRIPTION
**Pull request by kittysquee**
# [BUG: 404s for research sessions actually 500](https://trello.com/c/S08RNn5r)

I require some copy input and an email address for contact here, I will leave notes beside the sections which currently have dummy copy which will require updating.

Things this commit currently cover: 
* deleted standard Rails error pages
* routes for 404/500
* dealing with nested routes 404 

## 404 Page

screen shot 2018-01-22 at 12 01 31

## 500 Page

screen shot 2018-01-22 at 12 01 39

### CODE REVIEW

File: app/views/errors/internal_server_error.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1><%= t 'errors.internal_server_error' %></h1>
+<p>If you'd like to let us know there's been a problem, please <a href="mailto:me@email.com">email the team</a>.</p>
```
Comment (kittysquee) [inline app/views/errors/internal_server_error.html.erb]: This requires copy. It's the message which appears below the 500 page title. @sampling

File: app/views/errors/not_found.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1><%= t 'errors.not_found' %></h1>
+<p>If you'd like to let us know there's been a problem, please <a href="mailto:me@email.com">email the team</a>.</p>
```
Comment (kittysquee) [inline app/views/errors/not_found.html.erb]: This requires copy. It's the message which appears below the 404 page title. @sampling

File: config/locales/en.yml [hunk 3]
```diff
@@ -94,6 +94,8 @@ en:
 
   errors:
     format: '"%{attribute}" %{message}'
+    not_found: Sorry, the page you're looking for does not exist.
```
Comment (kittysquee) [inline config/locales/en.yml]: 404 page title -requires copy @sampling

File: config/locales/en.yml [hunk 4]
```diff
@@ -94,6 +94,8 @@ en:
...
+    internal_server_error: Sorry, we appear to have done something wrong.
```
Comment (kittysquee) [inline config/locales/en.yml]: 500 page title - require copy @sampling

**Comment by kittysquee** [comment]
This will need to be added to version 0.8.0 in the CHANGELOG.

File: config/locales/en.yml (hunk 4 above)
Comment (tosbourn) [reply config/locales/en.yml]: "Something has went wrong"

**Comment by tosbourn** [comment]
This is now ready for review.

File: app/views/errors/internal_server_error.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1><%= t 'errors.internal_server_error' %></h1>
+<p>If you'd like to let us know there's been a problem, please <a href="mailto:digital@barnardos.org.uk">email the team</a>.</p>
```
Comment (joelcarr) [inline app/views/errors/internal_server_error.html.erb]: Should we include the copy in `en.yml` too?

**Pull request merged**